heartbeat/data/heartbeat.pb.go

```diff
@@ -36,6 +36,7 @@ type Heartbeat struct {
 	Identity        string `protobuf:"bytes,7,opt,name=Identity,proto3" json:"Identity,omitempty"`
 	Pid             []byte `protobuf:"bytes,8,opt,name=Pid,proto3" json:"Pid,omitempty"`
 	Nonce           uint64 `protobuf:"varint,9,opt,name=Nonce,proto3" json:"Nonce,omitempty"`
+	PeerSubType     uint32 `protobuf:"varint,10,opt,name=PeerSubType,proto3" json:"PeerSubType,omitempty"`
```

Comment:
it appears this is the smallest unsigned int we can use in .proto files. :+1:  
Reference: https://developers.google.com/protocol-buffers/docs/proto3#scalar